### PR TITLE
fix: ensure proper overlay background layering

### PR DESCRIPTION
**Pull request by Niznikr**
## Summary

Ensure proper overlay background layering.

## Screenshots (if appropriate):

image-20240918-185348

### CODE REVIEW

**Review by apucacao — APPROVED** [review]
That's an Apple  quality graphic.

**Review by kwatkins-ld — APPROVED** [review]

File: packages/tokens/src/color-aliases.json [hunk 1]
```diff
@@ -183,10 +183,10 @@
 			"overlay": {
 				"primary": {
 					"$value": "{color.white.950}",
-					"dark": "{color.gray.900}"
+					"dark": "{color.gray.950}"
 				},
 				"secondary": {
-					"$value": "{color.gray.950}",
+					"$value": "{color.white.900}",
```
Comment (matthewferry) [inline packages/tokens/src/color-aliases.json:189]: hm, white 900 is semi transparent. I think maybe we mean to use `gray.0`? tho I'm not actually sure where we use secondary overlay tokens?

Comment (Niznikr) [reply packages/tokens/src/color-aliases.json:189]: Updated `Popover` to use secondary overlay since it should use its dark value.

Comment (Niznikr) [reply packages/tokens/src/color-aliases.json:189]: So had the value mirror how primary has white 950 and gray 950.

**Comment by Niznikr** [comment]
Closing in favor of the color refresh which will update the bg ui tokens to replace the overlay ones.

**Pull request closed without merging**